dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -167,6 +168,27 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
         }
     }
 
+    void AppNotificationManager::Register()
+    {
+        RegisterHelper(winrt::hstring{} /* displayName */, std::wstring{} /* iconFilePath */);
+    }
+
+    void AppNotificationManager::Register(hstring const& displayName, winrt::Uri const& iconUri)
+    {
+        THROW_HR_IF_MSG(E_ILLEGAL_METHOD_CALL, AppModel::Identity::IsPackagedProcess(), "Not applicable for packaged applications");
+
+        THROW_HR_IF(E_INVALIDARG, (displayName == winrt::hstring{}) || (iconUri == nullptr));
+
+        std::wstring iconFilePath{ iconUri.Path() };
+        iconFilePath = iconFilePath.substr(1, iconFilePath.size() - 1);
+        // Uri converts backward slashes to forward slashes
+        std::replace(iconFilePath.begin(), iconFilePath.end(), '/', '\\');
```

Comment:
You are actually converting forward slash to back slash. So the comment is incorrect. https://newbedev.com/difference-between-forward-slash-and-backslash-in-file-path